daprdocs/content/en/reference/components-reference/supported-state-stores/setup-azure-blobstorage.md

```diff
@@ -23,59 +22,123 @@ spec:
 
 | Field              | Required | Details | Example |
 |--------------------|:--------:|---------|---------|
 | accountName        | Y        | The storage account name | `"mystorageaccount"`.
-| accountKey         | Y        | Primary or secondary storage key | `"key"`
+| accountKey         | Y (unless using Azure AD) | Primary or secondary storage key | `"key"`
 | containerName      | Y         | The name of the container to be used for Dapr state. The container will be created for you if it doesn't exist  | `"container"`
-| ContentType        | N        | The blob’s content type | `"text/plain"`
+| `azureEnvironment` | N | Optional name for the Azure environment if using a different Azure cloud | `"AZUREPUBLICCLOUD"` (default value), `"AZURECHINACLOUD"`, `"AZUREUSGOVERNMENTCLOUD"`, `"AZUREGERMANCLOUD"`
+| ContentType        | N        | The blob's content type | `"text/plain"`
 | ContentMD5         | N        | The blob's MD5 hash | `"vZGKbMRDAnMs4BIwlXaRvQ=="`
 | ContentEncoding    | N        | The blob's content encoding | `"UTF-8"`
 | ContentLanguage    | N        | The blob's content language | `"en-us"`
 | ContentDisposition | N        | The blob's content disposition. Conveys additional information about how to process the response payload | `"attachment"`
 | CacheControl       | N        | The blob's cache control | `"no-cache"`
 
-## Setup Azure Blobstorage
+## Setup Azure Blob Storage
 
 [Follow the instructions](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal) from the Azure documentation on how to create an Azure Storage Account.
 
-If you wish to create a container for Dapr to use, you can do so beforehand. However, Blob Storage state provider will create one for you automatically if it doesn't exist.
+If you wish to create a container for Dapr to use, you can do so beforehand. However, the Blob Storage state provider will create one for you automatically if it doesn't exist.
 
 In order to setup Azure Blob Storage as a state store, you will need the following properties:
-- **AccountName**: The storage account name. For example: **mystorageaccount**.
-- **AccountKey**: Primary or secondary storage key.
-- **ContainerName**: The name of the container to be used for Dapr state. The container will be created for you if it doesn't exist.
+
+- **accountName**: The storage account name. For example: **mystorageaccount**.
+- **accountKey**: Primary or secondary storage account key.
+- **containerName**: The name of the container to be used for Dapr state. The container will be created for you if it doesn't exist.
+
+### Authenticating with Azure AD
+
+This component supports authentication with Azure AD as an alternative to use account keys. Whenever possible, it is reccomended that you use  Azure AD for authentication in production systems, to take advantage of better security, fine-tuned access control, and the ability to use managed identities for apps running on Azure.
```

Comment:
```suggestion
This component supports authentication with Azure AD as an alternative to use account keys. Whenever possible, it is recommended that you use  Azure AD for authentication in production systems, to take advantage of better security, fine-tuned access control, and the ability to use managed identities for apps running on Azure.
```

Reply:
I actually had suggested to simplify to "Whenever possible, [you should] use Azure AD to authenticate your Dapr components, for increased security..." to avoid the passive voice